### PR TITLE
feat: release 3.0.0

### PR DESCRIPTION
**Pull request by tzssangglass**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -62,6 +63,42 @@ title: Changelog
...
+- :sunrise: Support `session_secret` in openid-connect plugin to resolve the inconsistency of `session_secret` among multiple workers: [#8068](https://github.com/apache/apisix/pull/8068)
+- :sunrise: Support sasl config in kafka-logger plugin: [#8050](https://github.com/apache/apisix/pull/8050)
+- :sunrise: Support set resolve domain in proxy-mirror plugin: [#7861](https://github.com/apache/apisix/pull/7861)
+- :sunrise: Support `brokers` property in kafka-logger plugin, which supports different broker to set the same host: [#7999](https://github.com/apache/apisix/pull/7999)
+- :sunrise: Support get response body in ext-plugin-post-resp: [#7947](https://github.com/apache/apisix/pull/7947)
+- :sunrise: Added cas-auth plugin to support CAS authentication: [#7932](https://github.com/apache/apisix/pull/7932)
+
+### Bugfix
+
+- Conditional expressions of workflow plugins should support operators: [#8121](https://github.com/apache/apisix/pull/8121)
```
Comment (SylviaBABY) [inline CHANGELOG.md:97]:
```suggestion
- Conditional expressions of workflow plugin should support operators: [#8121](https://github.com/apache/apisix/pull/8121)
```

File: docs/zh/latest/CHANGELOG.md [hunk 2]
```diff
@@ -62,6 +63,42 @@ title: CHANGELOG
...
+
+- :sunrise: 新增 ai 插件，根据场景动态优化 APISIX 的执行路径：
+    - [#8102](https://github.com/apache/apisix/pull/8102)
+    - [#8113](https://github.com/apache/apisix/pull/8113)
+    - [#8120](https://github.com/apache/apisix/pull/8120)
+    - [#8128](https://github.com/apache/apisix/pull/8128)
+    - [#8130](https://github.com/apache/apisix/pull/8130)
+    - [#8149](https://github.com/apache/apisix/pull/8149)
+    - [#8157](https://github.com/apache/apisix/pull/8157)
+- :sunrise: openid-connect 插件支持设置 `session_secret`，解决多个 worker 间 `session_secret` 不一致：[#8068](https://github.com/apache/apisix/pull/8068)
```
Comment (SylviaBABY) [inline docs/zh/latest/CHANGELOG.md:88]:
```suggestion
- :sunrise: openid-connect 插件支持设置 `session_secret`，解决多个 worker 间 `session_secret` 不一致的问题：[#8068](https://github.com/apache/apisix/pull/8068)
```

File: CHANGELOG.md (hunk 1 above)
Comment (tzssangglass) [reply CHANGELOG.md:97]: workflow is name of a plugin

**Review by soulbird — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Review by spacewander — CHANGES_REQUESTED** [review]
We should also submit the new rockspec file, right?

**Comment by tzssangglass** [comment]
close this, replace with: https://github.com/apache/apisix/pull/8179

**Pull request closed without merging**